File: ios/Releaf/DesignSystem/AppSpacing.swift
/*
 * AppSpacing.swift + AppRadius.swift
 * Spacing and radius tokens. 4-pt grid for spacing; four radii.
 */

import CoreGraphics

public enum AppSpacing {
    public static let s0:  CGFloat = 0
    public static let s1:  CGFloat = 4
    public static let s2:  CGFloat = 8
    public static let s3:  CGFloat = 12
    public static let s4:  CGFloat = 16
    public static let s5:  CGFloat = 20
    public static let s6:  CGFloat = 24
    public static let s8:  CGFloat = 32
    public static let s10: CGFloat = 40
}

public enum AppRadius {
    public static let sm:   CGFloat = 6
    public static let md:   CGFloat = 12
    public static let lg:   CGFloat = 16
    public static let pill: CGFloat = 9999

    /// Floating BottomNav radius. Currently aliases `lg` (16pt).
    /// Swappable so shape changes to the floating nav stay in one place.
    public static let nav: CGFloat = lg
}

File: apps/releaf/android/app/src/main/java/app/releaf/mobile/ui/theme/AppSpacing.kt
/*
 * AppSpacing.kt + AppRadius.kt
 * 4-pt grid. Four radii.
 */

package app.releaf.mobile.ui.theme

import androidx.compose.ui.unit.dp

object AppSpacing {
    val s0  = 0.dp
    val s1  = 4.dp
    val s2  = 8.dp
    val s3  = 12.dp
    val s4  = 16.dp
    val s5  = 20.dp
    val s6  = 24.dp
    val s8  = 32.dp
    val s10 = 40.dp
}

object AppRadius {
    val sm   = 6.dp
    val md   = 12.dp
    val lg   = 16.dp
    val pill = 9999.dp

    /**
     * Floating BottomNav radius. Currently aliases `lg` (16dp). Swappable so
     * shape changes to the floating nav stay in one place.
     */
    val nav  = lg
}
